modules/ROOT/pages/salesforce/salesforce-composite-connector.adoc

```diff
@@ -30,7 +30,7 @@ You need login credentials to test your connection to your target resource.
 </dependency>
 ----
 
-Mule converts RELEASE to the latest version. To specify a version, view 
+Mule converts RELEASE to the latest version. To specify a version, view
```

Comment:
```suggestion
Mule converts `RELEASE` to the latest version. To specify a version, view
```
Is RELEASE syntax or is it a label or...? I'm just guessing here; maybe it does not require formatting? Where does it fit in the table of font choices in the style ref?